modules/editorial-metadata/editorial-metadata.php

```diff
@@ -61,7 +66,7 @@ public static function register_editorial_metadata_terms_as_post_meta(): void {
 		$editorial_metadata_terms = self::get_editorial_metadata_terms();
 
 		foreach ( $editorial_metadata_terms as $term ) {
-			$post_meta_key  = self::get_postmeta_key( $term );
+			$post_meta_key  = $term->meta[ self::METADATA_POSTMETA_KEY ];
```

Comment:
I think separating actual `WP_Term` data from supplied `->meta` is a good design choice. Currently custom statuses add all sorts of extras to the `WP_Term` directly, which can make it hard to tell what's built-in and what's dynamically added. We should probably do the same for custom statuses in the future.

Reply:
Yip, that was my intention with this. I didn't like how we were stuffing so much extra info in the description, when really that should have been going in as metadata.